app/views/shared/_posts.html.erb

```diff
@@ -1,37 +1,73 @@
-<% posts.each do |post| %>
-  <div class="post">
-    <h4>
-      <%= link_to post.title, post %>
-      <% if post.is_group %>
-        <div class="post__group_label label label-default"><%= I18n.t("activerecord.attributes.#{post.class.name.downcase}.is_group") %></div>
-      <% end %>
-    </h4>
-    <p>
-      <%= strip_tags(post.rendered_description.to_html) %>
-    </p>
-    <p>
-      <small>
-        <% if post.category %>
-          <%= glyph :folder_open %>
-          <span class="category">
-            <%= link_to [post.class, cat: post.category] do %>
-              <%= post.category %>
+<% posts.each_with_index do |post, postindex| %>
+  <div class="col-lg-6 col-md-6 col-sm-12">
+    <div class="zpost to-member-card">
+      <div class="to-member-card__header">      
+        <div class="to-member-card__header__text">
+          <h4>
+            <%= link_to post.title, post %>
+            <% if post.is_group %>
+              <div class="group">
+                <i class="material-icons u-va-6">people</i>
+                <%= I18n.t("activerecord.attributes.#{post.class.name.downcase}.is_group") %>
+              </div>            
             <% end %>
-          </span>
-        <% end %>
-        <% (post.tags || []).each do |tag| %>
-          <%= glyph :tag %>
-          <%= link_to [post.class, tag: tag] do %>
-            <%= tag %>
-          <% end %>
-        <% end %>
-      </small>
-    </p>
+          </h4>
+          <div class="to-member-card__header__text__activity">          
+            <%= link_to post.member do %>
+              <%= post.member %>
+            <% end %>            
+          </div>
+        </div>
+      </div>
+
+      <div class="to-member-card__body">
+        <div class="to-member-card__body__description">
+          <%= strip_tags(post.rendered_description.to_html) %>
+        </div>
+        
+        <div class="to-member-card__body__items row">
+          <div class="to-member-card__body__item col-lg-12 col-xs-12 u-p-15-xs" data-toggle="collapse" href="#collapseUserInfo-<%= post.id %>" role="button" aria-expanded="false" aria-controls="collapseUserInfo">
+            <i class="material-icons u-va-6 u-mr-20">keyboard_arrow_down</i>
+            <span class="translation_missing" title="translation missing: es.users.member_card.show_all">Show All</span>          
```

Comment:
Should add users.member_card.show_all key in /config/locales/en.yml in order to be translated

Reply:
Fixed